crates/swift-bridge-ir/src/parse/parse_extern_mod.rs

```diff
@@ -866,6 +890,25 @@ mod tests {
         );
     }
 
+    /// Verify that we push errors for unknown arguments
+    #[test]
+    fn error_args_into_arg_not_found() {
+        let tokens = quote! {
+            #[swift_bridge::bridge]
+            mod ffi {
+                extern "Rust" {
+                    #[swift_bridge(args_into = (foo, bar))]
+                    fn some_function(foo: u8);
+                }
+            }
+        };
+
+        let errors = parse_errors(tokens);
+
+        // Only "bar" should be missing argument.
+        assert_eq!(errors.len(), 1);
```

Comment:
Can we also match on `&errors[0]` and confirm that it is an `ArgsIntoArgNotFound` error?

Reply:
i.e. something like:

```rust
        assert_eq!(errors.len(), 1);
        match &errors[0] {
            ParseError::ArgsIntoArgNotFound {
                func: _,
                missing_arg,
            } => {
                assert_eq!(missing_arg, "bar")
            }
            _ => panic!(),
        };
```